Makefile

```diff
@@ -49,6 +49,7 @@ SERVICES :=     \
 SUBDIRS :=    \
 	x           \
 	cmd         \
+	collector   \
```

Comment:
interesting, no new services in the subrepo?

Reply:
Nope, that's coming with my new changes heh.

Reply:
Ah cheers